platform/admin/ui/src/main/webapp/js/views/Module.view.js

```diff
@@ -63,7 +63,11 @@ define([
                     tagName: 'div',
                     className: 'tab-pane',
                     onShow: function () {
-                        this.$el.find('iframe').iFrameResize();
```

Comment:
```
                   onAttach: function () {
                        this.$('iframe').ready(function(){
                            this.$('iframe').iFrameResize();
                        }.bind(this));
                    },
```

It's getting messed up because it's not actually in the dom yet at `onShow`.